armandolezama/practica-10-12-2019/SQLita/Administrative.js

```diff
@@ -0,0 +1,7 @@
+import { Employee } from "./Emplyee,js";
+
+export class Adminitrative extends Employee {
+    constructor(){
+        super();
```

Comment:
If you won't use the constructor in the child Class, you don't need instance the superclass